### DISCUSSION TITLE
Regarding "side sheets"

### DISCUSSION THREAD

**Discussion by nickpelone** [material-components/material-web#4313] [Q&A] [answered]
Apologies in advance, as this seems somewhat murky given available public info?

Original issue https://github.com/material-components/material-web/issues/477 tracked the `<mwc-drawer>` only supporting left-side drawers, and of course, it's been a while since then and we're dealing with a new Material spec. I think the spirit of the original issue was wanting a drawer to be able to be pulled out on the right. Like the issue mentions, custom CSS and etc can be injected, but I'm just sort of wondering what eventual plans are.

It appears instead as of Material 2 and then now 3, there's a [side sheet](https://m3.material.io/components/sheets-side/) thing to describe the "pop-out-able thing on the side for additional UI widgets" (I guess in this instance I'm referring to the modal versus standard form) or to otherwise describe a side-layout. It appears as [planned](https://m3.material.io/components/side-sheets/overview#:~:text=Available-,Implementation,-MDC%2DAndroid) on the implementation chart for Web, but there's no corresponding item in the readme (only near-neighbor being the "bottom sheet").

Sorry for the massive exposition, but basically: is a side-sheet that can be popped-out from the right side of the screen planned _eventually_ for the component library here? Or am I better served rolling my own for the foreseeable future?

Thanks in advance and for all the work done here! :)

**Comment by asyncliz** [comment, marked as answer]
Yes, it will be planned, but it likely won't happen for a while until after 1.0. I'd probably recommend rolling your own if you need one sooner.

**Reply by nickpelone** [reply]
Thanks for your response and everyone's efforts!
